installer/ansible/install_dependencies.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+echo Enabling universe repository
+sudo add-apt-repository universe
+
+echo Enabling docker repository
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
```

Comment:
Check docker & ansible install step from Delfin. "add-apt-repository" is getting depricated

Reply:
yes thanks. done